lib/libc/common/source/stdlib/malloc.c

```diff
@@ -251,6 +268,19 @@ void *realloc(void *ptr, size_t size)
 	ARG_UNUSED(ptr);
 	return malloc(size);
 }
+
+#ifdef CONFIG_SYS_HEAP_RUNTIME_STATS
+int malloc_runtime_stats_get(struct sys_memory_stats *stats)
```

Comment:
This might be more useful with a few changes.

Can you modify this to implement [mallinfo()] (https://man7.org/linux/man-pages/man3/mallinfo.3.html)?

Reply:
I could look into it. But I see that it has already been started to be implemented by #60269. Wouldn't I be redoing the same work ?

Reply:
Seems that way, but if you're feeling competitive, there is no reason not to try to get it in first 🥇